actions/contract-tests/action.yml

```diff
@@ -0,0 +1,36 @@
+name: "Contract Tests"
+description: 'Runs contract tests against an SDK component.'
+inputs:
+  repo:
+    required: false
+    description: "Name of the test harness repo"
+    default: 'sdk-test-harness'
+  version:
+    required: false
+    description: "Version of the test harness"
+    default: 'v2'
+  test_service_port:
+    required: true
+    description: "Port of the component under test."
+    default: '8123'
+  test_harness_port:
+    required: false
+    description: "Port of the test harness."
+    default: '8111'
+  debug_logging:
+    description: "Whether the test harness should print debug logs for failed tests."
+    required: false
+    default: 'false'
+  extra_params:
+    description: "Any other params to be passed to the test harness."
+    required: false
+
+runs:
+  using: composite
+  steps:
+    - name: 'Run Contract Tests'
+      shell: bash
+      run: curl -s https://raw.githubusercontent.com/launchdarkly/${{ inputs.repo }}/main/downloader/run.sh | bash
```

Comment:
We should modify this script to use the GITHUB_TOKEN when available. Sometimes it fails because we hit the anonymous api limit.